ckanext/spatial/harvesters/base.py

```diff
@@ -202,18 +203,21 @@ def get_package_dict(self, context, data_dict):
         :returns: A dataset dictionary (package_dict)
         :rtype: dict
         '''
-
+        
         tags = []
+
         if 'tags' in iso_values:
+            do_clean = self.source_config.get('clean_tags')
             for tag in iso_values['tags']:
-                tag = tag[:50] if len(tag) > 50 else tag
+                if do_clean:
```

Comment:
I would leave the original behaviour (truncate at 50 chars) if the `clean_tags` property is false.

Reply:
Fixed. For the record, munge_tag truncates to 100 chars (db field limit)